src/pages/api/hello.ts

```diff
@@ -1,10 +1,15 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { getAllPaths } from '@/utils/readFile'
 import type { NextApiRequest, NextApiResponse } from 'next'
 
 type Data = {
     name: string
+    data: any
 }
 
 export default function handler(req: NextApiRequest, res: NextApiResponse<Data>) {
-    res.status(200).json({ name: 'John Doe' })
+    const { search = '' } = req.query as any
```

Comment:
It is not a serious comment but I think I would have to change the name of `hello.ts` to `search.ts` or something like that later hehe